app/helpers/checks_helper.rb

```diff
@@ -1,23 +1,44 @@
+# frozen_string_literal: true
+
 module ChecksHelper
+  # Converts the HTTP status to the corresponding CSS class
+  # to control HTML element classes based on HTTP status
+  # @param status [Integer] the HTTP response status, `2XX..5XX`
```

Comment:
```suggestion
  # @param status [String] this is either the HTTP response status or an error, `2XX..5XX` or `"#{e}"`
```
@tareksamni 
status is a String which is either the HTTP response status or an error like 
* `Failed to open TCP connection to fail.brotandgames.net:443 (getaddrinfo: Name does not resolve)` or 
* `SSL_connect returned=1 errno=0 state=error: certificate verify failed (unspecified certificate verification error)`

This has to be changes throughout the PR.